src/Core/Framework/Plugin/Command/PluginListCommand.php

```diff
@@ -36,23 +41,35 @@ protected function configure(): void
     {
         $this
             ->setDescription('Show a list of available plugins.')
-            ->addOption('filter', 'f', InputOption::VALUE_REQUIRED, 'Filter the plugin list to a given term');
+            ->addOption('filter', 'f', InputOption::VALUE_REQUIRED, 'Filter the plugin list to a given term')
+            ->addOption(
+                'format',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Choose table or JSON format to output to',
```

Comment:
```suggestion
                'Choose "table" or "json" format to output to',
```

JSON as parameter would leak to an error :sweat_smile: